Domain/PsrAclCache.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Acl\Domain;
+
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\Security\Acl\Model\AclCacheInterface;
+use Symfony\Component\Security\Acl\Model\AclInterface;
+use Symfony\Component\Security\Acl\Model\ObjectIdentityInterface;
+use Symfony\Component\Security\Acl\Model\PermissionGrantingStrategyInterface;
+
+/**
+ * This class is a wrapper around a PSR-6 cache implementation.
+ *
+ * @author Michael Babker <michael.babker@gmail.com>
+ */
+class PsrAclCache implements AclCacheInterface
+{
+    use AclCacheTrait;
+
+    const PREFIX = 'sf_acl_';
+
+    private $cache;
+
+    /**
+     * @throws \InvalidArgumentException When $prefix is empty
+     */
+    public function __construct(CacheItemPoolInterface $cache, PermissionGrantingStrategyInterface $permissionGrantingStrategy, string $prefix = self::PREFIX)
+    {
+        if (0 === \strlen($prefix)) {
+            throw new \InvalidArgumentException('$prefix cannot be empty.');
+        }
+
+        $this->cache = $cache;
+        $this->permissionGrantingStrategy = $permissionGrantingStrategy;
+        $this->prefix = $prefix;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function clearCache()
+    {
+        $this->cache->clear();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function evictFromCacheById($aclId): void
+    {
+        $lookupKey = $this->getAliasKeyForIdentity($aclId);
+        $cacheItem = $this->cache->getItem($lookupKey);
+        if ($cacheItem->isHit()) {
+            $key = $cacheItem->get();
+            $this->cache->deleteItem($key);
+        }
+
+        $this->cache->deleteItem($lookupKey);
```

Comment:
```suggestion
        if (!$cacheItem->isHit()) {
            return;
        }

        $key = $cacheItem->get();
        $this->cache->deleteItem($key);
        $this->cache->deleteItem($lookupKey);
```

If we get a cache miss on the lookup item, there should be nothing we need to delete.

Reply:
Added the early return on there being no cache hit and then tweaked to a single `$this->cache->deleteItems()` call instead.